blackjack.py

```diff
@@ -17,10 +17,21 @@ class Card(object):
     def __init__(self, value, suit):
         self.value = value
         self.suit = suit
+        self.points = self.get_card_points()
 
     def __repr__(self):
         return '{} of {}'.format(self.value, self.suit)
 
+    def get_card_points(self):
+        """
+        Runs when a Card is instantiated
+        Assigns each card a points value
+        :return: int to be used for self.points attribute
+        """
+        return (10 if self.value in ('Jack', 'King', 'Queen')
```

Comment:
Side note for a future improvement, relying on strings like this can be tricky.
What if someone made a type and created a card with `jack` instead of `Jack`?
What if in the future you wanted to change things a bit and replace `Jack` with `Jesse`?

Instead, you should have constants, that would belong in the `Card` class to represent each of those strings and you'd refer to them that way.
For example:
```
class Card(object):
    JACK = 'Jack'
    KING = 'King'
    QWEEN = 'Qween'
    ...

    def get_card_points(self):
        return (10 if self.value in (self.JACK, self.KING, self.QWEEN) ...)
```

And when creating a card, you can do:
```
card = Card(Card.JACK, Card.HEARTS)
```